.changeset/sharp-pants-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+"react-magma-docs": patch
+---
+
+Feature/page templates
```

Comment:
Nitpick, but maybe this can be a little more descriptive, like `fix(PageTemplates): Updates to Page Templates examples, removed work in progress label.`

Reply:
Done. 😄